content/en/platform/corda/4.7/enterprise/node/node-flow-hospital.md

```diff
@@ -42,7 +42,7 @@ There are two aspects to the flow hospital:
 
 
 * run-time behaviour in the node upon failure, including retry and recovery transitions and policies.
-* visualisation of failed flows in the Explorer UI.
+* visualisation of failed flows in the [flow management console](../../../../../../en/platform/corda/4.7/enterprise/node/node-flow-management-console.md).
```

Comment:
Not sure we should change this in 4.7 since the conversion to American English only happened in 4.8. But I think we should change it in the 4.8 files for sure.

```suggestion
* vizualisation of failed flows in the [flow management console](../../../../../../en/platform/corda/4.7/enterprise/node/node-flow-management-console.md).
```